make-util.js

```diff
@@ -682,5 +693,270 @@ var stageTaskZipContent = function (sourceRoot, destRoot, metadataOnly) {
         });
     });
 }
-exports.stageTaskZipContent = stageTaskZipContent;
+
+var linkAggregatedLayoutContent = function (sourceRoot, destRoot, release, commit) {
+    assert(sourceRoot, 'sourceRoot');
+    assert(destRoot, 'destRoot');
+    assert(commit, 'commit');
+    console.log();
+    console.log(`> Linking ${path.basename(sourceRoot)}`);
+    mkdir('-p', destRoot);
+
+    // process each file/folder within the source root
+    fs.readdirSync(sourceRoot).forEach(function (itemName) {
+        // skip files
+        var taskSourcePath = path.join(sourceRoot, itemName);
+        if (!fs.statSync(taskSourcePath).isDirectory()) {
+            return;
+        }
+
+        // load the source task.json
+        var sourceTask = JSON.parse(fs.readFileSync(path.join(taskSourcePath, 'task.json')));
```

Comment:
The "package" target runs before "test" ("publish" runs after), so I think it's good to have some minimal light validation here (based on what's relevant for packaging process). Having thought about it more now, I would say it's good to have it to prevent failing the packaging process with a really strange error (i.e. end up in some weird unexpected state).

I believe there is a unit test to catch it, but i'm not sure whether it validates number vs string.